components/suggestion/switch_channel_provider.jsx

```diff
@@ -86,9 +85,7 @@ class SwitchChannelSuggestion extends Suggestion {
             const unreadMentions = collapsedThreads ? member.mention_count_root : member.mention_count;
             if (unreadMentions > 0) {
                 badge = (<div className={classNames('suggestion-list_unread-mentions', (isPartOfOnlyOneTeam ? 'position-end' : ''))}>
-                    <span
-                        className='badge'
-                    >
+                    <span className='badge'>
                         {unreadMentions}
                     </span>
                 </div>);
```

Comment:
please always follow this convention of having the component inside the brackets on new lines and indented. This makes it way easier to read.

I know that we do not have a eslint rule for that, but this is done all over the code-base in this manor.